modules/@apostrophecms/rich-text-widget/ui/apos/components/AposRichTextWidgetEditor.vue

```diff
@@ -130,52 +129,10 @@ export default {
     tiptapTypes() {
       return this.moduleOptions.tiptapTypes;
     },
-    // Enhances the dev-defined styles list with tiptap
-    // commands and parameters used internally.
-    styles() {
```

Comment:
this enhancement now happens as part of `editorOptions` computed, to keep everything in one place